dashboard/app/config.go

```diff
@@ -475,6 +496,20 @@ func checkKernelRepos(ns string, repos []KernelRepo) {
 			panic(fmt.Sprintf("%v: bad kernel repo reporting priority %v for %q", ns, prio, repo.Alias))
 		}
 		checkCC(&repo.CC)
+		for _, label := range []string{repo.LabelIntroduced, repo.LabelReached} {
+			if label == "" {
+				continue
+			}
+			if labelMap[label] {
```

Comment:
In the context of the other discussion: I think we need a global map of supported tags and check here that it's a recognized tag.